src/main/java/com/github/maxopoly/environmentaleffects/datarepresentations/MobConfig.java

```diff
@@ -155,6 +160,14 @@ public Location findSpawningLocation(Location loc) {
 							.contains(m)))
 							|| (spawnOnBlocks != null && spawnOnBlocks
 									.contains(m))) {
+						
+						for (Entity entity : loc.getWorld().getNearbyEntities(loc, minimumDistanceFromPlayer,
+								minimumDistanceFromPlayer, minimumDistanceFromPlayer)) {
+							if (entity instanceof Player) {
+								break;
```

Comment:
This only breaks out of the for loop, meaning you set bcs to FOUNDBASEBLOCK, leading to a mob spawn regardless if you find a player entity nearby or not.

Is that desired behavior?